src/markdown-pages/install-with-kurl/removing-minio.md

```diff
@@ -0,0 +1,71 @@
+---
+path: "/docs/install-with-kurl/removing-minio"
+date: "2021-12-17"
+weight: 21
+linktitle: "Removing Minio"
+title: "Removing Minio from your Cluster"
+---
+
+As of [v2021.12.20-0](https://kurl.sh/release-notes/v2021.12.20-0), kURL clusters can be installed without addons having a peer dependency on object storage. 
+There is also a migration path for clusters to remove object storage, such as is the case with removing Minio.
+
+# New Installations
+
+The following addons can be installed without object storage as noted:
+1. **Registry**: without an object store in the installer spec, a Persistent Volume (PV) will be used for storage. Note, that if the Rook  addon is installed, the Registry will ALWAYS use this as an object store.
```

Comment:
```suggestion
1. **Registry**: without an object store in the installer spec, a Persistent Volume (PV) will be used for storage. 

**Note:** The Registry will always use this as an object store if the Rook  add-on is installed.
```

Reply:
Can we keep this on the same line? I'm pretty sure the breaks will cause re-numbering of the list elements.

Reply:
```suggestion
1. **Registry**: without an object store in the installer spec, a Persistent Volume (PV) will be used for storage. **Note:** The Registry will always use this as an object store if the Rook add-on is installed.
```

Reply:
Good point! How about the above suggestion?

Reply:
@DanStough Is this specific to Rook because people might not realize it comes with an object store?

Reply:
I'm going to change `Rook` to object storage, since it covers both `Rook` and `Minio`, if that works for you?

Reply:
Yeah, that's fine.